src/components/core/text-field/text-field.tsx

```diff
@@ -0,0 +1,450 @@
+import zxcvbn from 'zxcvbn';
```

Comment:
We should lazy load `zxcvbn` library since this is a huge package. We only use Password Meter at a few places. So,  I think it's better to lazy load.

Reply:
Will incorporate this suggestion